internal/pkg/featureflags/feature_flags.go

```diff
@@ -42,13 +44,15 @@ const (
 // Manager is a global feature flag manager
 var Manager featureFlagManager
 
-var attributes = []string{"user.resource_id", "org.resource_id", "environment.id", "cli.version", "cluster.id", "cluster.physicalClusterId"}
+var attributes = []string{"user.resource_id", "org.resource_id", "environment.id", "cli.version", "cluster.id", "cluster.physicalClusterId", "cli.command", "cli.flags"}
 
 type featureFlagManager interface {
 	BoolVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal bool) bool
 	StringVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal string) string
 	IntVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal int) int
 	JsonVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal any) any
+
+	SetCommandAndFlags(cmd *cobra.Command, args []string)
```

Comment:
No reason to add this here... I don't think we're mocking this function.

Reply:
I need to add this here so the function is visible in internal/cmd/command.go, as Manager is of type featureFlagManager.